src/test/java/com/alibaba/wasp/jdbc/TestJdbcStatement.java

```diff
@@ -180,5 +183,6 @@ public void testStatement() throws SQLException, IOException,
 
     admin.close();
     stat.close();
+    }
   }
-}
\ No newline at end of file
+}
```

Comment:
Please remove unnecessary changes. You may have added an empty line at the end of this file when there wasn't one.

Reply:
I don't know why, but the GitHub editor is weirdly adding a line automatically at the end of the file. Even if I remove the line, it appears again. It appears in the 'Files changed' page but doesn't appear when I just view the commit.